backend/npdfhir/tests.py

```diff
@@ -160,8 +161,9 @@ class BasicViewsTestCase(APITestCase):
     def test_health_view(self):
         url = reverse("healthCheck")  # maps to "/healthCheck"
         response = self.client.get(url)
+        res_obj = json.loads(response.content.decode())
```

Comment:
it would be cleaner to [use the built-in](https://docs.djangoproject.com/en/5.2/topics/testing/tools/#django.test.Response.json) `response.json()` here, which does the same thing.

```suggestion
        res_obj = response.json()
```

Reply:
made this change